extension/src/plots/model/index.ts

```diff
@@ -168,9 +173,9 @@ export class PlotsModel extends ModelWithPersistence {
       ...Object.keys(this.revisionData)
     ])
 
-    return this.getSelectedRevisions().filter(
-      revision => !cachedRevisions.has(revision)
-    )
+    return this.getSelectedRevisions()
+      .filter(label => !cachedRevisions.has(label))
+      .map(label => this.getCLIId(label))
```

Comment:
[F] Whenever we get the missing revisions to call the CLI with we replace the branch label with the short sha.